docs/reference/migrations-tool-configuration.md

```diff
@@ -109,4 +109,21 @@ The username that will be used to connect to the ksqlDB server. A username and
 
 The password that will be used to connect to the ksqlDB server. A 
 [username](#ksqlauthbasicusername) and password will be passed as part of 
-HTTP basic authentication. 
\ No newline at end of file
+HTTP basic authentication. 
+
+Migrations Directory Configs
+----------------------------
+
+### `ksql.migrations.dir.override`
```

Comment:
We should specify that the migration tool should be on version 0.25+

Reply:
Good call. I think this is shipping with 0.26 so that's what I put in the docs. I'll update separately if the 0.25 cut gets delayed and this manages to get picked up.